### PR TITLE
Add mobile menu

### PR DESCRIPTION
**Pull request by ntimanu**
In this project we:

- Added JavaScript
- Added mobile menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -16,50 +16,83 @@ body {
...
 }
 
-.portfolio,
-.about,
-.contactmenu {
-  display: none;
-}
-
-.navcontainer {
+.navbar {
```
Comment (MahmoudBakr23) [inline style.css:27]: - [x] Please align the menu button as shown in the original design.

Screenshot (1)

File: style.css [hunk 2]
```diff
@@ -16,50 +16,83 @@ body {
...
+  margin-top: 24px;
+  margin-left: 20px;
+  font-family: Poppins, sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 18px;
   line-height: 20px;
+}
+
+.navbar-links {
```
Comment (MahmoudBakr23) [inline style.css:49]: - [x] Kindly work on the width issue with the mobile menu

screen2

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @ntimanu😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 
There are some excellent points that you have made perfectly. Keep up! 💪

- Well done with the design so far
- Descriptive PR and README file
- Correct Linters

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (ntimanu) [reply style.css:27]: Done

File: style.css (hunk 2 above)
Comment (ntimanu) [reply style.css:49]: Done

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello Team! 👋 
### [OPTIONAL]

- [ ] There are two grey vertical lines on line on the left and right of the mobile menu. Kindly work on it and remove it. Mobile Menu should stick in the center which can be done by the following example. 👍 
#### For Example
```
max-width: 375px;
margin: 0 auto;
```
Screenshot 2023-03-09 at 2 00 21 PM


## Things to Appreciate 
- [x] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [x] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**